earth_enterprise/rpms/build.gradle

```diff
@@ -0,0 +1,138 @@
+buildscript {
+    repositories {
+        maven {
+            url "https://plugins.gradle.org/m2/"
+        }
+    }
+    dependencies {
+        classpath "com.netflix.nebula:gradle-ospackage-plugin:4.4.2"
+    }
+}
+
+apply plugin: "nebula.ospackage"
+
+def env = System.getenv()
+def home = env['HOME']
+def tmp_install = '/tmp/fusion_os_install'
```

Comment:
I think we should use a build directory in the package building directory tree.  We don't want to interfere with whatever build options a developer may have used to build Open GEE.

Reply:
Also, I'd call this something like `build_install_root`, so it's a bit more descriptive. The same goes for the variables below.

Reply:
Also, some typically do mount /tmp with tmpfs...this may be something we want to change in general...